iset.mm

```diff
@@ -105767,549 +105767,549 @@ which defines the set we are taking the supremum of must (a) be true at
-    oddpwdc.j $e |- J = { z e. NN | -. 2 || z } $.
-    oddpwdc.f $e |- F = ( x e. J , y e. NN0 |-> ( ( 2 ^ y ) x. x ) ) $.
-    $( The function ` F ` that decomposes a number into its "odd" and "even"
-       parts, which is to say the largest power of two and largest odd divisor
-       of a number, is a bijection from pairs of a nonnegative integer and an
-       odd number to positive integers.  (Contributed by Thierry Arnoux,
-       15-Aug-2017.) $)
-    oddpwdc $p |- F : ( J X. NN0 ) -1-1-onto-> NN $=
-      ( cn0 cn wtru c2 cv cexp co cdvds wbr wn wa wcel wceq va cxp wf1o cmul c1
-      caddc crio cdiv cc cr 2cnd simpr expcld breq2 notbid elrab2 simplbi nncnd
-      adantr mulcld adantl nnnn0 2nn wreu pw2dvdseu riotacl syl nnexpcl sylancr
-      nn0nndivcl syl2anc jca wb anbi1i oddpwdclemdc bitri a1i f1od2 trud ) EHUB
-      IDUCJABUAEHKBLZMNZALZUDNZIDUALZKKCLZMNWDOPKWEUEUFNMNWDOPQRZCHUGZMNZUHNZWG
-      UIUJHGWBESZVTHSZRZWCUISJWLWAWBWLKVTWLUKWJWKULUMWLWBWJWBISZWKWJWMKWBOPZQZK
-      WEOPZQWOCWBIEWEWBTWPWNWEWBKOUNUOFUPZUQUSURUTVAWDISZWIUJSZWGHSZRJWRWSWTWRW
-      DHSWHISZWSWDVBWRKISWTXAVCWRWFCHVDWTCWDVEWFCHVFVGZKWGVHVIWDWHVJVKXBVLVAWLW
-      DWCTZRZWRWBWITVTWGTRRZVMJXDWMWORZWKRZXCRXEWLXGXCWJXFWKWQVNVNCWDWBVTVOVPVQ
-      VRVS $.
+    $d K z $.  $d N z $.
+    algfx.6 $e |- ( z e. S -> ( ( C ` z ) = 0 -> ( F ` z ) = z ) ) $.
+    $( If ` F ` reaches a fixed point when the countdown function ` C ` reaches
+       ` 0 ` , ` F ` remains fixed after ` N ` steps.  (Contributed by Paul
+       Chapman, 22-Jun-2011.) $)
+    ialgfx $p |- ( A e. S -> ( K e. ( ZZ>= ` N ) ->
+        ( R ` K ) = ( R ` N ) ) ) $=
+      ( wcel cfv wceq wi vm vk cz cuz cn0 ffvelrni syl5eqel nn0zd wa cv cle wbr
+      crab uzval eleq2d pm5.32i c1 caddc co fveq2 eqeq1d imbi2d eqidd a1i sylan
+      eluznn0 cc0 nn0uz 0zd id ialgrp1 syldan ialgrf ffvelrnda ialgcvga eqeq12d
+      wf imp imbi12d vtoclga sylc eqtrd biimprd expcom adantl sylbir a2d uzind3
+      sylbi ex com3r mpd ) BEQZHUCQZGHUDRZQZGDRZHDRZSZTWMHWMHBCRUENEUEBCLUFUGZU
+      HWNWPWMWSWNWPWMWSTZWNWPUIWNGHAUJZUKULAUCUMZQZUIXAWNWPXDWNWOXCGAHUNZUOUPWM
+      UAUJZDRZWRSZTWMWRWRSZTZWMUBUJZDRZWRSZTWMXKUQURUSZDRZWRSZTXAUAAUBHGXFHSZXH
+      XIWMXQXGWRWRXFHDUTVAVBXFXKSZXHXMWMXRXGXLWRXFXKDUTVAVBXFXNSZXHXPWMXSXGXOWR
+      XFXNDUTVAVBXFGSZXHWSWMXTXGWQWRXFGDUTVAVBXJWNWMWRVCVDWNXKXCQZUIZWMXMXPYBWN
+      XKWOQZUIWMXMXPTZTZWNYCYAWNWOXCXKXEUOUPYCYEWNWMYCYDWMYCUIZXPXMYFXOXLWRYFXO
+      XLFRZXLWMYCXKUEQZXOYGSWMHUEQYCYHWTXKHVFVEZWMBDEFXKVGIUEVHKWMVIZWMVJZEEFVQ
+      WMJVDZEIQWMOVDZVKVLYFXLEQZXLCRZVGSZYGXLSZWMYCYHYNYIWMUEEXKDWMBDEFVGIUEVHK
+      YJYKYLYMVMVNVLWMYCYPABCDEFXKHIJKLMNOVOVRXBCRZVGSZXBFRZXBSZTYPYQTAXLEXBXLS
+      ZYSYPUUAYQUUBYRYOVGXBXLCUTVAUUBYTYGXBXLXBXLFUTUUBVJVPVSPVTWAWBVAWCWDWEWFW
+      GWHWIWJWKWL $.
   $}
 
+
 $(
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-                                 Algorithms
+        Rationality of square root of 2
```

Comment:
Non-rationality of the square root of 2 (?)

Reply:
I'll gladly accept this section header if it leads to people not complaining about the header. The idea is to cover both non-rationality and irrationality in this section, although the latter won't be fully developed until all this number theory stuff is in place.